src/clojure_tictactoe/game/game_play.clj

```diff
@@ -1,34 +1,32 @@
 (ns clojure-tictactoe.game.game-play
-  (:require   [clojure-tictactoe.cli.output.board-printer :as board-printer]
+  (:require [clojure-tictactoe.cli.output.board-printer :as board-printer]
+            [clojure-tictactoe.cli.output.instructions-printer :as instructions-printer]
             [clojure-tictactoe.cli.input.input-getter :as input-getter]
+            [clojure-tictactoe.cli.output.end-game-printer :as end-printer]
             [clojure-tictactoe.game.players :as players]
             [clojure-tictactoe.game.board :as board]
             [clojure-tictactoe.game.game-rules :as rules]
-            [clojure-tictactoe.cli.output.end-game-printer :as end-printer]
             ))
 
-(defn player-move
-  [move-function board]
-  (loop []
-    (if-let [new-board (board/mark-space (move-function) @players/player-atom board)]
-      new-board
-      (do (println "Sorry, that looks taken, try again") 
-          (recur)))))
+(defn player-move [move-function board]
+  (board/mark-space (move-function board) @players/player-atom board))
 
-(defn continue-game
-  [board]
-    (let [new-board (player-move input-getter/get-player-choice board)]
-       (board-printer/print-board new-board)
-       (players/player-swap)
-       new-board))  
+(defn continue-game [board]
+  (let [new-board (player-move (players/choose-player-function @players/player-atom) board)]
+    (board-printer/print-board new-board)
+    (players/player-swap)
+    new-board))
 
-(defn game-loop
-  ([]
-   (game-loop (board/render-empty-board)))
-  ([board]
-   (if-not (rules/game-over? board)
-      (do (let [new-board (continue-game board)]
-        (recur new-board)))
-      (do (let [results (rules/assess-win board)]
-            (end-printer/print-game-over results))))))
+(defn game-loop []
+  (instructions-printer/print-game-intro)
+  (input-getter/continue-to-game)
+  (let [board (board/render-empty-board 9)]
+    (board-printer/print-board board)
```

Comment:
What you're doing above here is not strictly `game-loop`. Maybe extract them out into an `initialize-game` function that returns a `board` you can pass to `game-loop`.

Reply:
👍